### PR TITLE
added vendor tail analytics (#30815)

### PR DESCRIPTION
**Pull request by babadopulos**
@micajuine-ho  
Adding tail analytics vendor support

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors/tail.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "request": "pageview"
+    }
+  },
+  "transport": {
+    "beacon": false,
+    "xhrpost": false,
+    "image": {
+      "suppressWarnings": true
+    }
+  }
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/vendors/tail.json:20]: AMP-analytics will try beacon -> xhrpost -> image requests and handle the fallback. Just to confirm, your servers only accept image requests?

**Review by micajuine-ho — COMMENTED** [review]
Hi @babadopulos thank you for your integration! 

Just one question.

File: extensions/amp-analytics/0.1/vendors/tail.json (hunk 1 above)
Comment (babadopulos) [reply extensions/amp-analytics/0.1/vendors/tail.json:20]: Hi @micajuine-ho,

you are right, we only accept image requests yet.

**Review by micajuine-ho — APPROVED** [review]

**Comment by micajuine-ho** [comment]
@babadopulos Approved, just need you to sign the CLA (above)

**Comment by babadopulos** [comment]
@micajuine-ho signed

**Pull request merged**